src/services/builder/types.ts

```diff
@@ -34,3 +38,41 @@ export interface Proposal {
   voteStart: string
   voteEnd: string
 }
+
+export interface Attestation {
+  recipient: string
+  decodedDataJson: string
+  timeCreated: number
+}
+
+export interface AttestationJsonData {
+  name: string
+  type: string
+  signature: string
+  value: {
+    name: string
+    type: string
+    value: string | number
+  }
+}
+
+export interface PropdateObject {
+  propId: number
+  replyTo: string
+  response: string
+  milestoneId: number
+}
+
+export interface Propdate extends PropdateObject {
+  chain: Chain
+  recipient: string
+  timeCreated: number
+}
+
+export interface UpdateDao {
```

Comment:
```suggestion
export interface DaoMetadata {
```

Name is somewhat misleading, its not an update to the dao but some metadata of the dao & proposal.